.changeset/clever-cars-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": minor
```

Comment:
I'd argue this is a major change. `Util` is exported from `index.ts`:

``` TS
export {default as Util} from "./util";
```

You'll see that this breaks Webapp because I used `Util.keScoreFromPerseusScore` in my recent change. So:

1. Probably a major change
2. I'd double check the utils that you moved to make sure they're not used in Webapp (and export the ones that are)

Reply:
Agreed. I didn't catch the one usage of `keScoreFromPerseusScore` outside of Perseus.  Fixing.